code/game/jobs/access.dm

```diff
@@ -236,12 +236,13 @@ var/obj/item/card/id/all_access/ghost_all_access
 /mob/living/bot/GetIdCard()
 	return botcard
 
-/mob/living/carbon/human/GetIdCard()
-	var/obj/item/I = get_active_hand()
-	if(I)
-		var/id = I.GetID()
-		if(id)
-			return id
+/mob/living/carbon/human/GetIdCard(var/in_hand = TRUE)
+	if(in_hand)
```

Comment:
Don't need the in_hand argument here since if an ID is in the hands then we should check for that last and the ID slot first.